.tekton/on-pull-request.yaml

```diff
@@ -4,14 +4,12 @@ kind: PipelineRun
 metadata:
   name: vulnerability-analysis-on-pr
   annotations:
-    # The event we are targeting as seen from the webhook payload
-    # this can be an array too, i.e: [pull_request, push]
-    pipelinesascode.tekton.dev/on-event: "[pull_request]"
-
-    # The branch or tag we are targeting (ie: main, refs/tags/*)
-    pipelinesascode.tekton.dev/on-target-branch: "[main, rh-aiq-main]"
-    # Paths that trigger the pipeline on change
-    pipelinesascode.tekton.dev/on-path-change: "[src/**, metrics_lib/**, pyproject.toml, Dockerfile, .dockerignore]"
+    # Trigger on non-draft PRs to target branches when specific paths are changed.
+    pipelinesascode.tekton.dev/on-cel-expression: |
+      event == "pull_request" &&
+      !body.pull_request.draft &&
+      (target_branch == "main" || target_branch == "rh-aiq-main") &&
+      ("src/**".pathChanged() || "metrics_lib/**".pathChanged() || "pyproject.toml".pathChanged() || "Dockerfile".pathChanged() || ".dockerignore".pathChanged())
```

Comment:
Hi @vbelouso 
I Missing only the path of `uv.lock` here, which should also trigger the pipeline.